internal/util/experimental/experimental.go

```diff
@@ -64,5 +64,6 @@ func (e Experimental) AppendNotice(description *string) {
 }
 
 var (
-	DNS = New("DNS API", "in beta", "https://docs.hetzner.cloud/changelog#2025-10-07-dns-beta")
+	DNS        = New("DNS API", "in beta", "https://docs.hetzner.cloud/changelog#2025-10-07-dns-beta")
+	StorageBox = New("Storage Box Integration", "experimental", "TODO") // TODO: Create and link ticket before release
```

Comment:
```suggestion
	StorageBox = New("Storage Box implementation", "experimental", "TODO") // TODO: Create and link ticket before release
```

Sounds better. I am not sure if we should stick to the 2 strings (product and maturity), we could use a free text argument, ensuring we cover most if not all cases.

Reply:
I will call it `Storage Box support` to match the CLI.